src/DropdownKebab/__snapshots__/DropdownKebab.test.js.snap

```diff
@@ -16,6 +16,7 @@ exports[`Kebab dropdown renders properly 1`] = `
     type="button"
   >
     <span
+      aria-hidden={true}
```

Comment:
It looks like this snapshot test went from being shown to hidden?  Do we not have a snapshot test for the dropdown shown case?

Reply:
+1 I noticed the same. The react-fontawesome library used is small enough that I think it would be simple enough to incorporate our own implementation without it (i.e.: it's one component here: https://github.com/danawoodman/react-fontawesome/blob/master/src/index.js). Depending on @jgiardino's feedback, I may self implement it (as it looks like currently the `aria-hidden` is hard set to `true`.